includes/csharp10-templates.md

```diff
@@ -0,0 +1,10 @@
+---
+author: billwagner
+ms.author: wiwagn
+ms.date: 08/20/2021
+ms.topic: include
+---
+> [!IMPORTANT]
```

Comment:
I wrote this to be accurate for preview 7. When the RC 1 changes roll out, we can update this include to the new behavior going forward.